src/benchq/problem_ingestion/molecule_instance_generation.py

```diff
@@ -117,12 +179,7 @@ def truncate_with_avas(
         mean_field_object, loc_type="pm"
     )  # default is loc_type ='pm' (Pipek-Mezey)
 
-    # Truncates to a specific active space that captures the essential chemistry
-    molecule, mean_field_object = avas_active_space(
-        mean_field_object, ao_list=ao_list, minao=minao
-    )
-
-    return mean_field_object
+    return avas_active_space(mean_field_object, ao_list=ao_list, minao=minao)
```

Comment:
Add a return type perhaps?

Reply:
Fixed in 549744d2e9d2e0ef8ba2353841fbe8eb69c229cd